src/Symfony/Component/PropertyAccess/Annotation/ConfigurationAnnotation.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Base configuration annotation.
+ *
+ * @author Johannes M. Schmitt <schmittjoh@gmail.com>
```

Comment:
I didn't make any significant change to the Serializer component borrowed code (just changed the class name and the namespace) so the credit is all yours :)

I'll add mine too then.